backend/authorization_service/src/bin/authorization-service.ts

```diff
@@ -0,0 +1,5 @@
+import * as cdk from 'aws-cdk-lib';
+import { AuthorizationServiceStack } from '../lib/authorization-service-stack';
+
+const app = new cdk.App();

```

Comment:
Why do you have a separate application for each service?
As you have one BE application, it's totally make sense create this one application with multiple stacks

https://docs.aws.amazon.com/cdk/v2/guide/stack_how_to_create_multiple_stacks.html

Reply:
@KunskayaV 
to simulate microservice architecture